src/App.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import resume from './data/resume.json';
+
+import { Awards, Basics, Interests, Languages, Publications, References, Skills, Experience } from './components';
+
+import './App.scss';
+
+function App() {
+  return (
+    <div className="App">
+      {resume.basics && <Basics data={resume.basics} />}
+      {resume.work && <Experience data={resume.work} type='work' orgType='company' title='Work Experience' />}
+      {resume.volunteer && <Experience data={resume.volunteer} type='volunteer' orgType='organization' title='Volunteer' />}
+      {resume.education && <Experience data={resume.education} type='education' orgType='institution' title='Education' />}
+      
+      <div className='app-row'>
+        {resume.awards && <Awards data={resume.awards} />}
+        {resume.publications && <Publications data={resume.publications} />}
+      </div>
+      
+      {resume.skills && <Skills data={resume.skills} />}
+      
+      <div className='app-row'>
+        {resume.languages && <Languages data={resume.languages} />}
+        {resume.interests && <Interests data={resume.interests} />}
+      </div>
+      
+      {resume.references && <References data={resume.references} />}
```

Comment:
We can do an `Object.entries` to convert the object into an array of [key, value] per each object key.
So in case the resume adds new keys will be easier to render new components.

Code example:
```javascript
const resumeEntries = Object.entries(resume)

return (
<div className="App"
  {
     resumeEntries.map(([key, data]) => {
       switch (key): {
         case 'basics': return <Basics data={data} />
         case 'work': return <Experience data={data} type="work" orgType="company" />
         case 'volunteer': return <Experience data={data} type="work" orgType="volunteer" />
         case 'education': return <Experience data={data} type="work" orgType="volunteer" />
         // and so on..
       }
     })
  }
</div>
)
```

Link:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

Reply:
I really like the Experience component great idea!